optimum/bettertransformer/models/base.py

```diff
@@ -61,6 +63,13 @@ def __init__(self, config):
                 self.num_layers = getattr(config, attr)
                 break
 
+        if old_layer is not None:
+            # Last step, store the old module skeleton by copying the old module and putting
+            # it on the `meta` device.
+            self.old_layer = deepcopy(old_layer).to("meta")
```

Comment:
Do we need to make a deepcopy?

Reply:
To be safe it's better I guess but not sure